ouster_ros/src/motion_correction_node.cpp

```diff
@@ -0,0 +1,111 @@
+#include <memory>
+
+#include <minkindr_conversions/kindr_tf.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <pcl_ros/point_cloud.h>
+#include <ros/ros.h>
+#include <sensor_msgs/Imu.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <tf/transform_listener.h>
+
+#include "ouster_ros/motion_correction.h"
+
+namespace ouster_ros {
+namespace OS1 {
+
+class MotionCorrectionNode {
+ public:
+  void tfTimerCallback(const ros::TimerEvent&) {
```

Comment:
Do you think there's benefits in the future of adding a transform callback as well, similar to what we have in voxblox? We'd have to put some transforms/calibrations in...

Reply:
It might be nice to have, but it also complicates things quite a bit. I was trying to keep it simple for now and build on if needed